1_fetch/src/get_feature_vars.R

```diff
@@ -399,7 +469,7 @@ prep_feature_vars <- function(sb_var_data, sites_all, sites_screened,
            year = str_sub(name, 14, 17)) %>%
     group_by(COMID, unit) %>%
     summarise(avg_annual = mean(value, na.rm = TRUE), .groups = "drop") %>%
-    mutate(label = paste0(unit, "_AVG_WILDFIRE_shorterm_avg")) %>%
+    mutate(label = paste0(unit, "_WILDFIRE_shortterm_avg")) %>%
```

Comment:
I think this will cause downstream modeling targets to be outdated. For now, I think we should keep as we had it before.
```suggestion
    mutate(label = paste0(unit, "_AVG_WILDFIRE_shorterm_avg")) %>%
```

Reply:
yeah you are right... so I guess I will revert the HAYPASTURE to Hay_PASTURE and ICESNOW to ICE_SNOW as well

Reply:
You can file an issue to rename these columns. That can be done with the reorganization of the column order